src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt

```diff
@@ -3229,6 +3405,67 @@ class LicenceServiceTest {
       verify(auditEventRepository, never()).saveAndFlush(any())
       verify(licenceEventRepository, never()).saveAndFlush(any())
     }
+
+    @Test
+    fun happyPathWhenVaryingTimeServedLicenceWithNoComAllocated() {
```

Comment:
can we call this `Activating time served licence variation with no responsible COM allocated`?